infra/stack.py

```diff
@@ -360,29 +353,6 @@ def add_service_client(
             user_pool_client_name=f"{service_id} Service Access",
             disable_o_auth=False,
         )
-        # temp: we are going provide client id, secret, and user pool id values twice in the secret (once with veda_ prefix)
-        service_client_secret = self._get_client_secret(client)
-        cognito_app_secret = self._create_secret(
-            service_id,
-            {
-                "flow": "client_credentials",
-                "cognito_domain": self.domain.base_url(),
-                "client_id": client.user_pool_client_id,
-                "client_secret": service_client_secret,
-                "userpool_id": self.userpool.user_pool_id,
-                "veda_client_id": client.user_pool_client_id,
-                "veda_client_secret": service_client_secret,
-                "veda_userpool_id": self.userpool.user_pool_id,
-                "scope": " ".join(scope.scope_name for scope in scopes),
-            },
-        )
-        stack_name = Stack.of(self).stack_name
-        CfnOutput(
-            self,
-            f"cognito-app-{service_id}-secret",
-            export_name=f"{stack_name}-cognito-app-secret",
```

Comment:
this is the secret we're passing to `veda-data-airflow`, no?

Reply:
so airflow uses the veda-workflows auth client? I will duplicate this output in app.py. The other app.py cfn output that was added for the programmatic client was causing duplicate export names so I removed this inner cfn output to let the new one you added for sdk win.

Reply:
Oh that won't work either. I may need to walk back some changes and try and handle the duplicate identifier bug in a different way.

Reply:
yeah, it uses this export name - `{stack_name}-cognito-app-secret`

Reply:
I think I fixed this here [7adb071](https://github.com/NASA-IMPACT/veda-auth/pull/24/commits/7adb071dbb3021bc33d5f225603331feea008724)